test_weather_alexander.py

```diff
@@ -0,0 +1,215 @@
+import time
+
+import pytest
+import requests
+import json
+from requests import get
+import jsonschema
+from jsonschema import validate
+import os
+
+
+# TODO: test other endpoints
+# TODO: add parametrization by the city
+
+
+@pytest.mark.parametrize('name_city, name_country, ', [
+    ('London', 'GB'),
+    ('Moscow', 'RU'),
+    ('Taganrog', 'RU'),
+])
+def test_weather(name_city, name_country):
+    url = 'http://api.openweathermap.org/data/2.5/weather?q='+name_city+','+name_country+'&appid='
```

Comment:
use params argument to pass URL query parameters:
```python
get(url, params={'q': name_city+','+name_country, 'appid': key})
```

Reply:
Done